enterprise_gateway/services/sessions/kernelsessionmanager.py

```diff
@@ -93,6 +93,7 @@ def create_session(self, kernel_id, **kwargs):
         # Compose the kernel_session entry
         kernel_session = dict()
         kernel_session["kernel_id"] = kernel_id
+        kernel_session["tenant_id"] = self.kernel_manager.tenant_kernels.get_tenant_id(kernel_id)
```

Comment:
can we read from `kwargs.get("env", {}).get("KERNEL_TENANT_ID")` ?

Reply:
We're able to just use `km.tenant_id` since it will have that value set at this point.